### ISSUE TITLE
Happymh: Blocked from website/Unable to load image

### ISSUE THREAD

**Issue by Wcraft233** [keiyoushi/tachiyomi-extensions-source#1913]
### Source information

Happymh 1.4.8

### Source language

Chinese

### Steps to reproduce

1.use UA
`Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Safari/537.36 QIHU 360ENT`
2.Blocked from website
3.Use mobile browser UA
4.Can access web page but cannot load images

### Expected behavior

Visit the website and load the image

### Actual behavior

Blocked by site or unable to load image

### Mihon/Tachiyomi version

0.16.4

### Android version

Android 13

### Other details

Can you provide a UA that can access the website and load the images?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Nevermind425** [comment]
Do you find any ways to slove this?I have already changed my sauce for now

**Comment by raine93** [comment]
Hi, change to this user agent and it works

https://github.com/keiyoushi/extensions-source/issues/1409

**Comment by Nevermind425** [comment]
No it doesn't useable anymore

**Comment by raine93** [comment]
> No it doesn't useable anymore

Hmm that's weird..I just used it today a few times and it worked

**Comment by Nevermind425** [comment]
Bruh really?Can you me more details,such as your settings cause it doesn't work for 
me anymore

**Comment by Nevermind425** [comment]
![Screenshot_2024-03-24-03-15-01-058_app mihon](https://github.com/keiyoushi/extensions-source/assets/101038505/be057714-2e32-4858-baf5-5ecced9c83e2)

**Comment by Smol-Ame** [comment]
Just a reminder, just cause a user-agent works for one person doesn't mean it'll work for you/another. Go search for another user-agent, maybe a modern/latest Chinese browser one.

**Comment by Nevermind425** [comment]
Honestly I just want to know is that my personal problem or not only me have this problem .If I can't find any ways to slove this,I will change my sauce.

**Comment by Nevermind425** [comment]
Oh I find that when I using vpn I can use this sauce again , but I am not using this UA:Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Safari/537.36 QIHU 360ENT
I am using the newest Chrome UA
